python/tank/bootstrap/manager.py

```diff
@@ -583,11 +585,41 @@ def get_pipeline_configurations(self, project):
             current_login=self._sg_user.login,
             sg_connection=self._sg_connection,
         ):
+            try:
+                path = ShotgunPath.from_shotgun_dict(pc)
+                config_path = path.current_os
+            except TankError:
+                log.warning(
+                    "Unable to determine config path for %s: %s",
```

Comment:
On a completely unrelated topic, how do we deal when caching actions with pipeline app store or git descriptors that don't specify the version? In the resolver he query the app store to fetch the latest. Do we need to do the same when we return descriptors here?